fvp.mk

```diff
@@ -135,7 +135,8 @@ BR2_ROOTFS_POST_SCRIPT_ARGS = "$(FVP_VIRTFS_AUTOMOUNT) $(FVP_VIRTFS_MOUNTPOINT)"
 # ARM Trusted Firmware
 ################################################################################
 TF_A_EXPORTS ?= \
-	CROSS_COMPILE="$(CCACHE)$(AARCH64_CROSS_COMPILE)"
+	CROSS_COMPILE="$(AARCH64_CROSS_COMPILE)" \
```

Comment:
Why not the same thing as in `qemu_v8.mk`?
```
TF_A_EXPORTS ?= \
        CROSS_COMPILE="$(CCACHE)$(AARCH64_CROSS_COMPILE)" \
        CC="$(CCACHE)$(AARCH64_CROSS_COMPILE)gcc" \
        LD="$(CCACHE)$(AARCH64_CROSS_COMPILE)ld"
```

Reply:
I suspect that's because you didn't include `$(CCACHE)` in `CROSS_COMPILE.

Reply:
Can you please explain? According to my understanding adding ccache to `CROSS_COMPILE` does not work since TF-A v2.12, and this is fixed here. According to that I could not have ccache in CROSS_COMPILE.

Reply:
Anyways I added the LD line. TF-A seems to call LD trough the compiler wrapper (gcc) and I don't think this has any effect, but it does no harm, so let's have it for consistency.